content/chainguard/libraries/verification.md

```diff
@@ -44,10 +44,52 @@ The following requirements must be met:
 
 ## Access 
 
-chainver is available to customers upon request. The archive includes binaries
-for different operating systems and processor architectures.
+[Download the latest release - version 0.3.5](https://dl.enforce.dev/chainver/0.3.5/chainver-v0.3.5.zip)
+
+Use the following script to automatically determine the latest available version
+and download the ZIP archive.
+
+```shell
+# Get the latest version
+export LATEST=$(curl -s "https://storage.googleapis.com/us.artifacts.prod-enforce-fabc.appspot.com/?prefix=chainver/" | \
+  grep -oE 'chainver/[0-9]+\.[0-9]+\.[0-9]+/' | \
+  sed 's|chainver/||g' | sed 's|/$||g' | \
+  sort -V | tail -1)
+# Download the release zip file
+curl -LO "https://dl.enforce.dev/chainver/${LATEST}/chainver-v${LATEST}.zip"
+```
+
+Extract the ZIP archive and find archives for different operating systems and
+processor architectures in the created `chainver-package/archives` directory:
+
+```
+chainver_0.3.5_Linux_x86_64.tar.gz
+chainver_0.3.5_Darwin_arm64.tar.gz
+chainver_0.3.5_Darwin_x86_64.tar.gz
+chainver_0.3.5_Linux_arm64.tar.gz
+chainver_0.3.5_Windows_x86_64.zip
+```
+
+Extract the package, in the example for MacOS and ARM processor, and copy it to
+a directory that is on the `PATH`:
+
+```shell
```

Comment:
This is not ignored by precommit hook @ltagliaferri

Reply:
I updated to ignore code blocks but let me know if there's still an issue